pkg/envoy/bootstrap/config.go

```diff
@@ -41,6 +41,29 @@ func BuildFromConfig(config Config) (*xds_bootstrap.Bootstrap, error) {
 		return nil, err
 	}
 
+	tlsMinVersion := xds_transport_sockets.TlsParameters_TLSv1_2
+	if config.TLSMinimumProtocolVersion != "" {
```

Comment:
Please move this logic to retrieve and set the TLS version on the transport_socket to it's own helper function, with unit tests.

Reply:
Updated.

FYI, we need to keep this default values in MeshConfig schema rather than change it to Envoy default value. TLS_AUTO breaks our mesh connection.

Reply:
Default value is declare in MeshConfig schema